<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabExperiment">
       <attribute name="title">
        <string>&amp;Experiment</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLog">
       <attribute name="title">
        <string>&amp;Log</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPlot">
       <attribute name="title">
        <string>&amp;Plot</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="CPlotWidget" name="plotWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>X</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboX">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Y</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboY">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>PSize</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="pointSizeSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>15</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QSlider" name="pointSizeSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>15</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>First</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="firstSpinBox"/>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QSlider" name="firstSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Last</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSpinBox" name="lastSpinBox"/>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="QSlider" name="lastSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="QCheckBox" name="sampleWeightCheckBox">
              <property name="text">
               <string>Sample Weight</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0" colspan="2">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="9" column="0" colspan="2">
             <widget class="QCheckBox" name="mergeReplicationsCheckBox">
              <property name="text">
               <string>Merge Replications</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMatrix">
       <attribute name="title">
        <string>&amp;Hessian</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableWidget" name="matrixTable">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEigen">
       <attribute name="title">
        <string>Ei&amp;genvectors</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTableWidget" name="eigenTable">
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMax">
       <attribute name="title">
        <string>&amp;Max</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QTableWidget" name="maxTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWinRate">
       <attribute name="title">
        <string>Win Rate</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QTableWidget" name="winRateTable"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Close"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About_Qt"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Experiment">
    <property name="title">
     <string>&amp;Experiment</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="action_SoftStop"/>
    <addaction name="action_HardStop"/>
    <addaction name="separator"/>
    <addaction name="action_Automatic_GUI_update"/>
    <addaction name="action_Update_GUI_manually_now"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Experiment"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="action_SoftStop">
   <property name="text">
    <string>S&amp;oft Stop (wait for running games to finish)</string>
   </property>
   <property name="toolTip">
    <string>Soft Stop</string>
   </property>
  </action>
  <action name="action_HardStop">
   <property name="text">
    <string>&amp;Hard Stop (kill everything)</string>
   </property>
   <property name="toolTip">
    <string>Hard Stop</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="action_Automatic_GUI_update">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Automatic GUI update</string>
   </property>
  </action>
  <action name="action_Update_GUI_manually_now">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Update GUI manually now</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CPlotWidget</class>
   <extends>QWidget</extends>
   <header>CPlotWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>comboX</tabstop>
  <tabstop>comboY</tabstop>
  <tabstop>pointSizeSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboY</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>plotWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>firstSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>firstSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pointSizeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>plotWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>plotWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lastSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lastSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboX</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>plotWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>firstSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>firstSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pointSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>firstSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>plotWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sampleWeightCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>plotWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Automatic_GUI_update</sender>
   <signal>toggled(bool)</signal>
   <receiver>action_Update_GUI_manually_now</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Update_GUI_manually_now</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Automatic_GUI_update</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateConnections()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeReplicationsCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>plotWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateAll()</slot>
  <slot>updateConnections()</slot>
 </slots>
</ui>
